cmd/generate-analytics-api.rb

```diff
@@ -122,7 +122,10 @@ def generate_analytics_api
         DAYS.each do |days|
           next if days != "30" && category_name == "build-error" && !data_source.nil?
 
-          output = run_formula_analytics(*formula_analytics_args, "--days-ago=#{days}")
+          args = %W[--days-ago=#{days}]
+          args << "--influxdb" if days == "30"
```

Comment:
This shouldn't trigger together with `--Linux `

Reply:
Since we don't even have a full 30 day window I want to do the 30 days first. If we have a little more (maybe 60) I'd say we can move to InfluxDB completely.

Reply:
@SMillerDev sounds good!